Main.hs

```diff
@@ -55,7 +55,7 @@ runKritiInteractive (KritiOptions jsonFile templateFile rootSymbol) = do
   void $ checkFilePath templateFile
 
   json <- LBS.readFile jsonFile
-  template <- B.readFile templateFile
+  template <- TIO.readFile templateFile
```

Comment:
Docs for `TIO.readFile` warn that this is locale-dependent: https://hackage.haskell.org/package/text-2.0/docs/Data-Text-IO.html#v:readFile so I don't think we should be using it here.

I think maybe we still need a version of `runKriti` that directly takes a `ByteString`.

I.e.
```haskell
runKritiBS :: B.ByteString -> [(T.Text, J.Value)] -> Either RenderedError J.Value
runKritiBS = ...

runKriti :: T.Text -> [(T.Text, J.Value)] -> Either RenderedError J.Value
runKriti = runKritiBS . T.encodeUtf8
```

Then `main` can call `runKritiBS` with a `ByteString` read directly from the file. Other callers that want to pass the template as `Text` can call `runKriti`.